### DISCUSSION TITLE
<title>Your account is marked as spam<title>

### DISCUSSION THREAD

**Discussion by Whe3r** [community/community#114420] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have a problem with my account. It has been marked as spam because I created a second account. However, I cannot delete one of my accounts. GitHub simply does not let me do this. I have written to support and they have responded to me, but they have not provided any information on how to solve the problem. I am at a loss and do not know what to do. Can someone please help me?

**Comment by benogoulart** [comment]
Hi,

If your account has been marked as spam, the restriction itself can only be reviewed by GitHub Support. Unfortunately, the community can't remove spam flags or access account details.

Regarding the duplicate account:

- If GitHub is preventing you from deleting one of the accounts because of the restriction, only GitHub Support can resolve that issue.
- Since you've already contacted Support, I recommend replying to your existing ticket rather than opening new ones, and clearly explain that you're unable to delete the duplicate account due to the current restriction.

Could you clarify what happens when you try to delete the account?

- Do you receive an error message?
- Or is the **Delete account** option unavailable?

That information may help determine whether it's a restriction-related issue or a different problem.
